app/helpers/guides_helper.rb

```diff
@@ -2,4 +2,24 @@ module GuidesHelper
   def book_an_appointment_link?
     !@guide.related_to_appointments?
   end
+
+  def previous_guide
+    return @previous_guide if defined?(@previous_guide)
```

Comment:
Actually scratch what I just said. I see what you mean. Although I think the added clarity is worth the extra passes through these methods, personally. WDYT?